tg_searcher/common.py

```diff
@@ -48,6 +48,8 @@ class CommonBotConfig:
     @staticmethod
     def _parse_proxy(proxy_str: str):
         url = url_parse.urlparse(proxy_str)
+        if url.username and url.password:
+            return url.scheme, url.hostname, url.port, True, url.username, url.password
```

Comment:
The `True` parameter is the rdns flag in PySocks. It could be less confused if adding an comment for this.

Reply:
Done.